app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+# Gem 'bcrypt-ruby'
+# Generate User model with password_digest field
+class User < ApplicationRecord
+  # belongs_to :role
+  # belongs_to :level
+  # has_and_belongs_to_many :inventory
+  # has_and_belongs_to_many :skills
+
+  has_secure_password
+
+  before_save { self.email = email.downcase }
```

Comment:
please use separate `method` not a block in before_save.

Reply:
fix